src/libstd/net/ip.rs

```diff
@@ -89,6 +89,9 @@ impl Ipv4Addr {
     }
 
     /// Returns true if this is a loopback address (127.0.0.0/8).
+    ///
+    /// This property is defined by RFC 6890
```

Comment:
Nah I think that if you're reading this much into the documentation you'll probably recognize in the context that these are the other kind of RFCs. That being said if it ends up being confusing we can certainly update!